applications/GeoMechanicsApplication/custom_conditions/U_Pw_condition.cpp

```diff
@@ -63,7 +63,8 @@ void UPwCondition<TDim,TNumNodes>::
     }
     else
     {
-        KRATOS_THROW_ERROR(std::logic_error, "undefined dimension in U_Pw_condition!!", "" );
+        std::string ErrorMessage = "undefined dimension in U_Pw_condition!!";
+        KRATOS_ERROR << ErrorMessage << std::endl;
```

Comment:
```suggestion
        KRATOS_ERROR << "undefined dimension in U_Pw_condition!!" << std::endl;
```

this is the intended usecase